### DISCUSSION TITLE
Do we need to use Microsoft recommended driver block rules when Windows already uses them by default?

### DISCUSSION THREAD

**Discussion by HotCakeX** [MicrosoftDocs/WDAC-Toolkit#217] [Q&A] [answered]
[With Windows 11 2022 update, the vulnerable driver blocklist is enabled by default for all devices.](https://learn.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules).

When creating a policy, WDAC Policy wizard app has an option to integrate Microsoft recommended block rules with the policy we are creating.

Is it necessary to do it when HVCI and Memory Integrity are already turned on in Windows (Which means Windows uses the recommended driver block rules)? it's important to know this because I don't want to be enforcing the recommended driver block rules in two places at the same time.

Thanks

**Comment by jgeurten** [comment, marked as answer]
If you only manage Windows 11 22H2 systems, then you don't need the recommended driver block rules in your WDAC policy. Otherwise, you should have the driver block rules in your policy. 

In either scenario, you should have the recommended user mode rules

**Reply by HotCakeX** [reply]
Thank you very much!
